languagetool-core/src/main/java/org/languagetool/rules/AbstractCheckCaseRule.java

```diff
@@ -131,6 +141,14 @@ public RuleMatch[] match(AnalyzedSentence sentence) {
     return toRuleMatchArray(ruleMatches);
   }
 
+  public boolean ignoreShortUppercaseWords() {
+    return ignoreShortUppercaseWords;
+  }
+
+  public void setIgnoreShortUppercaseWords(boolean value) {
+    ignoreShortUppercaseWords = value;
+  }
+
   private boolean isPunctuationStart(String word) {
```

Comment:
You are overwriting the method `public boolean ignoreShortUppercaseWords()`, but in the code you are using the variable `ignoreShortUppercaseWords`, which is unchanged by your method. 
Remove one of these methods, and use only the other one.